app/helpers/http_client.js

```diff
@@ -0,0 +1,16 @@
+const axios = require('axios');
+const logger = require('../logger');
+
+const { httpClientError } = require('../errors');
+
+// eslint-disable-next-line arrow-body-style
```

Comment:
Good job to define a httpClient method is a good idea. I think that `return` is not necessary you should return the promise directly for example 
`const httpClient = (method, url, data = '') =>
    axios({ method, url, data })
    .then(response => response.data)
    .catch(error => {
      logger.error(`httpClient Error => ${error.message}`);
      httpClientError(`httpClient Error => ${error.message}`);
    });
`